modin/experimental/batch/test/test_pipeline.py

```diff
@@ -409,7 +409,7 @@ def new_col_adder(df, partition_id):
             NotImplementedError,
             match="Dynamic repartitioning is currently only supported for DataFrames with 1 partition.",
         ):
-            new_dfs = pipeline.compute_batch()
+            _ = pipeline.compute_batch()
```

Comment:
```suggestion
            pipeline.compute_batch()
```

Why assign anything at all?